src/Telegram.Bot/Types/Update.cs

```diff
@@ -119,20 +136,22 @@ public class Update
     /// </value>
     public UpdateType Type => this switch
     {
-        { Message: { } }            => UpdateType.Message,
-        { EditedMessage: { } }      => UpdateType.EditedMessage,
-        { InlineQuery: { } }        => UpdateType.InlineQuery,
-        { ChosenInlineResult: { } } => UpdateType.ChosenInlineResult,
-        { CallbackQuery: { } }      => UpdateType.CallbackQuery,
-        { ChannelPost: { } }        => UpdateType.ChannelPost,
-        { EditedChannelPost: { } }  => UpdateType.EditedChannelPost,
-        { ShippingQuery: { } }      => UpdateType.ShippingQuery,
-        { PreCheckoutQuery: { } }   => UpdateType.PreCheckoutQuery,
-        { Poll: { } }               => UpdateType.Poll,
-        { PollAnswer: { } }         => UpdateType.PollAnswer,
-        { MyChatMember: { } }       => UpdateType.MyChatMember,
-        { ChatMember: { } }         => UpdateType.ChatMember,
-        { ChatJoinRequest: { } }    => UpdateType.ChatJoinRequest,
-        _                           => UpdateType.Unknown
+        { Message: { } }              => UpdateType.Message,
+        { EditedMessage: { } }        => UpdateType.EditedMessage,
+        { InlineQuery: { } }          => UpdateType.InlineQuery,
+        { ChosenInlineResult: { } }   => UpdateType.ChosenInlineResult,
+        { CallbackQuery: { } }        => UpdateType.CallbackQuery,
+        { ChannelPost: { } }          => UpdateType.ChannelPost,
+        { EditedChannelPost: { } }    => UpdateType.EditedChannelPost,
+        { ShippingQuery: { } }        => UpdateType.ShippingQuery,
+        { PreCheckoutQuery: { } }     => UpdateType.PreCheckoutQuery,
+        { Poll: { } }                 => UpdateType.Poll,
+        { PollAnswer: { } }           => UpdateType.PollAnswer,
+        { MyChatMember: { } }         => UpdateType.MyChatMember,
+        { ChatMember: { } }           => UpdateType.ChatMember,
+        { ChatJoinRequest: { } }      => UpdateType.ChatJoinRequest,
+        { MessageReaction: { } }      => UpdateType.MessageReaction,
+        { MessageReactionCount: { } } => UpdateType.MessageReactionCount,
```

Comment:
Can you replace `{ }` to `not null` in each arm like so `{ Message: not null }`? It reads more explicitly like that.